app/controllers/organizations/staff/fosterers_controller.rb

```diff
@@ -1,9 +1,42 @@
 class Organizations::Staff::FosterersController < Organizations::BaseController
   layout "dashboard"
+  before_action :set_fosterer, only: %i[update_activation]
 
   def index
     authorize! AdopterFosterAccount, context: {organization: Current.organization}
 
     @fosterer_accounts = authorized_scope(AdopterFosterAccount.fosterers)
   end
+
+  def deactivate
+    @fosterer_account.deactivate
+    respond_to do |format|
+      format.html { redirect_to staff_fosterers_index_path, notice: "Fosterer account deactivated." }
```

Comment:
Can we use translations here? (en.yml)
Also, can we display a flash for turbostream?

Reply:
I can certainly try